python/mlx/nn/layers/base.py

```diff
@@ -146,7 +146,7 @@ def load_weights(
         self,
         file_or_weights: Union[str, List[Tuple[str, mx.array]]],
         strict: bool = True,
-    ):
```

Comment:
Why quote Module here? Can you not just do `-> Module` directly?

Reply:
No. If I do `-> Module` or `-> self`, since the class is not defined completely yet, it gives error `Module is not defined`. So, we should use python's "forward referencing" that is mentioning the class name inside quotes

Reply:
I see, that makes sense.